transformato/charmm_factory.py

```diff
@@ -327,7 +316,7 @@ def _get_CHARMM_waterbox_postprocessing_body(self, env: str):
             self.configuration["simulation"].get("GPU", False) == True
             and self.charmm_gpu == "domdec-gpu"
         ):
-            GPU_domdec = "domdec gpu on"
+            GPU_domdec = "domdec gpu only"
```

Comment:
are you sure about this?

Reply:
Yes, you can use both, but according to Stefan the option `GPU only` is slightly faster...